br/cmd/br/main.go

```diff
@@ -49,6 +49,7 @@ func main() {
 		NewBackupCommand(),
 		NewRestoreCommand(),
 		NewStreamCommand(),
+		newOpeartorCommand(),
```

Comment:
```suggestion
		NewOpeartorCommand(),
```

Reply:
I think this shouldn't be public function. 🤔️